marker/v2/schema/registry.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict, Type, TypeVar
+
+from marker.v2.schema import BlockTypes
+from marker.v2.schema.blocks import Block, Caption, Code, Equation, Figure, \
+    Footnote, Form, Handwriting, InlineMath, \
+    ListItem, PageFooter, PageHeader, Picture, \
+    SectionHeader, Table, TableOfContents, \
+    Text
+from marker.v2.schema.document import Document
+from marker.v2.schema.groups import FigureGroup, ListGroup, PageGroup, \
+    PictureGroup, TableGroup
+from marker.v2.schema.text import Line, Span
+
+BLOCK_REGISTRY: Dict[str, Type[Block]] = {
```

Comment:
I think this will be safer for multiprocessing:

```
from importlib import import_module
BLOCK_REGISTRY = {}

def register_block(key, block_class):
    BLOCK_REGISTRY[key] = f"{block_class.__module__}.{block_class.__name__}"

def get_block_class(block_type):
    class_path = BLOCK_REGISTRY[block_type]
    module_name, class_name = class_path.rsplit('.', 1)
    module = import_module(module_name)
    return getattr(module, class_name)
```

Reply:
done!